tests/tests.uischema

```diff
@@ -8,7 +8,8 @@
         "triggers",
         "generator",
         "selector",
-        "schema"
+        "schema",
+        "dialogs"
```

Comment:
Changes to this file are most likely not intentional.

Reply:
Correct, I'll revert this change as it's not intended.